### ISSUE TITLE
[BUG] prsgui fails to start

### ISSUE THREAD

**Issue by WorldTeacher** [KnoBuddy/VisualDiffusion#12]
**Describe the bug**
when executing prsgui.py, it fails to start and displays this error message:
```
Prompts loaded
Traceback (most recent call last):
  File "VisualDiffusion/prsgui.py", line 608, in <module>
    set_variables()
  File "VisualDiffusion/prsgui.py", line 105, in set_variables
    n_samples = json_set['n_samples']
KeyError: 'n_samples'

```

**To Reproduce**
1. Install progrockstable
2. Install VisualDiffusion
3. enter python VisualDiffusion/prsgui.py

**Expected behavior**
The GUI should open

**Desktop (please complete the following information):**
 - OS: Arch Linux (Kernel: 5.19.5)
 - GPU and VRAM: 3090 24GB
 - CUDA Version: 11.7

**Comment by KnoBuddy** [comment]
This is a product of ripping out the grid and collage stuff. I'll get right on it.

**Comment by KnoBuddy** [comment]
Fixed in /pull/13.

Please let me know if you run into any further issues.

**Comment by WorldTeacher** [comment]
I did a quick test and everything works fine, will do more extensive tests tomorrow and finally test stable-diffusion ^^
Thanks for the quick fix
